llvm/test/CodeGen/AIE/aie2/end-to-end/TahnTemplated-swp.ll

```diff
@@ -0,0 +1,340 @@
+  %90 = tail call noundef <16 x bfloat> @llvm.aie2.v16accfloat.to.v16bf16(<8 x i64> %89)
+  %91 = tail call <32 x bfloat> @llvm.aie2.set.bf512.bf256(<16 x bfloat> %90, i32 0)
+  %92 = tail call <32 x bfloat> @llvm.aie2.upd.bf512.bf256(<32 x bfloat> %91, <16 x bfloat> %13, i32 1)
+  %93 = tail call noundef <8 x i64> @llvm.aie2.bf.mul16.conf(<32 x bfloat> %92, <32 x bfloat> %15, i32 60)
+  %94 = tail call noundef <8 x i64> @llvm.aie2.sub.accfloat(<8 x i64> %93, <8 x i64> %62, i32 28)
+  %95 = tail call noundef <16 x bfloat> @llvm.aie2.v16accfloat.to.v16bf16(<8 x i64> %94)
+  store <16 x bfloat> %64, ptr addrspace(6) %p_out.0, align 32, !tbaa !11
+  %add.ptr.i7.i.i = getelementptr inbounds i8, ptr %p_out.0.in17, i20 32
+  %add.ptr.ascast.i8.i.i = addrspacecast ptr %add.ptr.i7.i.i to ptr addrspace(6)
+  store <16 x bfloat> %95, ptr addrspace(6) %add.ptr.ascast.i8.i.i, align 32, !tbaa !11
+  %add.ptr.i9.i = getelementptr inbounds i8, ptr %p_out.0.in17, i20 64
+  %96 = call i1 @llvm.loop.decrement.i32(i32 1)
+  br i1 %96, label %for.body, label %for.cond.cleanup, !llvm.loop !12
+}
+
+; Function Attrs: nocallback nofree nosync nounwind willreturn memory(none)
+declare { <32 x bfloat>, i32 } @llvm.aie2.vmin.gebf16(<32 x bfloat>, <32 x bfloat>) #2
+
+; Function Attrs: nocallback noduplicate nofree nosync nounwind willreturn
+declare void @llvm.set.loop.iterations.i32(i32) #5
+
+; Function Attrs: nocallback noduplicate nofree nosync nounwind willreturn
+declare i1 @llvm.loop.decrement.i32(i32) #5
+
+attributes #0 = { nounwind memory(none) }
+attributes #1 = { nounwind memory(inaccessiblemem: read) }
+attributes #2 = { nocallback nofree nosync nounwind willreturn memory(none) }
+attributes #3 = { nocallback nofree nosync nounwind willreturn memory(inaccessiblemem: read) }
+attributes #4 = { mustprogress noinline "no-builtin-memcpy" "no-jump-tables"="true" "no-trapping-math"="true" "stack-protector-buffer-size"="8" }
+attributes #5 = { nocallback noduplicate nofree nosync nounwind willreturn }
+
+!llvm.linker.options = !{}
+!llvm.module.flags = !{!0, !1, !2}
+!llvm.ident = !{!3}
+
+!0 = !{i32 7, !"Dwarf Version", i32 4}
+!1 = !{i32 2, !"Debug Info Version", i32 3}
+!2 = !{i32 1, !"wchar_size", i32 4}
+!3 = !{!"clang version 19.0.0git (/scratch/llvm-aie/clang 640962db16e997d4aaf9dadcf09d9a4fc7e06fe4)"}
+!4 = !{!5, !6, i64 0}
+!5 = !{!"TanhTemplated", !6, i64 0, !7, i64 4, !7, i64 5, !7, i64 6, !7, i64 7, !7, i64 8, !7, i64 9, !9, i64 10, !6, i64 12, !6, i64 16, !6, i64 20, !6, i64 24, !6, i64 28, !10, i64 32}
+!6 = !{!"int", !7, i64 0}
+!7 = !{!"omnipotent char", !8, i64 0}
+!8 = !{!"Simple C++ TBAA"}
+!9 = !{!"short", !7, i64 0}
+!10 = !{!"_ZTS23tanh_templated_params_tIu6__bf16E"}
+!11 = !{!7, !7, i64 0}
+!12 = distinct !{!12, !13, !14}
+!13 = !{!"llvm.loop.mustprogress"}
+!14 = !{!"llvm.loop.itercount.range", i64 4}
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
+; WAW-STICKY-ON-PIPELINER: {{.*}}
```

Comment:
Can the check lines still be auto generated?

Reply:
I think no, sadly! If you agree I can keep just the instructions, not the pipeliner info.

Reply:
I simplified the test! Now it can be ;-)